gateways/istio-egress/templates/deployment.yaml

```diff
@@ -20,6 +20,10 @@ spec:
     matchLabels:
       app: istio-egressgateway
       istio: egressgateway
+  strategy:
```

Comment:
Should this be protected by an if as well ? Doesn't seem to be something we want in 100% of cases.

Reply:
why wouldn't?  For demo cases we don't care about upgrade?

Reply:
Good point. We actually do care probably, but no reason it wouldn't work.